simulation_pose_topic_name: /sim/center_pose

gps_info_topic_name: /drivers/gps

localization_utm_topic_name: /localization/utmpose

node_rate: 50   # [Herz]


